packages/ui/src/components/NavBar.tsx

```diff
@@ -0,0 +1,43 @@
+import Link from 'next/link'
+import Logo from './Logo'
+
+interface Props {
+  tabs: { name: string; href: string }[]
+  playerName: string
+  playerLevel: number
+  playerHref?: string
+}
+
+function NavBar({
```

Comment:
Alright, it was just easier to have this. I will leave it like this until we replace it. -> I think it makes sense to define a new task (otherwise it might be forgotten ^^), wdys?